media/js/firefox/facebook-container-video.js

```diff
@@ -0,0 +1,42 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+(function() {
+    'use strict';
+
+    var $html = $(document.documentElement);
+    var videoPoster = new Mozilla.VideoPosterHelper('.fbcontainer-video');
+
+    if (window.Mozilla.Client.isFirefox) {
+        $html.addClass('is-firefox');
+    } else {
+        $html.addClass('is-not-firefox');
+    }
```

Comment:
Doesn't look like this `else` condition is used - can we remove it?

Reply:
Removed!